src/test/utilities/childProcess.test.ts

```diff
@@ -102,8 +88,24 @@ describe('ChildProcess', async () => {
 
             const result = await childProcess.promise()
 
-            assert.strictEqual(result.exitCode, 0)
-            assert.strictEqual(result.stdout, 'hi')
+            assert.strictEqual(result.exitCode, 0, `Expected exit code 0, got ${result.exitCode}`)
+            assert.strictEqual(result.stdout, 'hi', `Expected stdout to be hi , got: ${result.stdout}`)
+        })
+
+        it('runs commands containing a space - unix', async () => {
```

Comment:
To avoid duplicating code (which could become out of sync in the future), suggest to combine the two tests like:

```typescript
const scriptFile = path.join(tempFolder, `test script.${os.platform === 'win' ? 'bat' : 'sh'}`)
```